net/message/transaction.go

```diff
@@ -32,14 +31,7 @@ type trn struct {
 func (msg trn) Handle(node Noder) error {
 	log.Debug()
 	log.Debug("RX Transaction message")
-	tx := &msg.txn
-	if !node.LocalNode().ExistedID(tx.Hash()) {
-		if err := va.VerifyTransaction(tx); err != nil {
-			return errors.New("[VerifyTransaction] error")
-		}
-		if err := va.VerifyTransactionWithLedger(tx, ledger.DefaultLedger); err != nil {
-			return errors.New("[VerifyTransactionWithLedger] error")
-		}
+	if !node.LocalNode().ExistedID(msg.txn.Hash()) {
 		node.LocalNode().AppendTxnPool(&(msg.txn))
```

Comment:
The verification functions shouldn't  wrap into the appendTxnPool which have no relationship by watching the function name